hermes-api/src/main/java/pl/allegro/tech/hermes/api/SentMessageTrace.java

```diff
@@ -142,6 +142,15 @@ public boolean equals(Object obj) {
             return false;
         }
         final SentMessageTrace other = (SentMessageTrace) obj;
-        return Objects.equals(this.messageId, other.messageId) && Objects.equals(this.subscription, other.subscription);
+        return Objects.equals(this.messageId, other.messageId)
```

Comment:
same here, i bet noone needs all of them to be equal. don't make test code/reqeuirements leak into the main